base/utils/R/convert.input.R

```diff
@@ -115,17 +115,16 @@ convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, st
       # regular expression specific special characters and site-specific identification is already present through
       # site.id, a regex placeholder is used instead.
       
-      filename_pattern = pattern #pattern is always the name of the meteorological data product (met).
-      if (!is.null(input.args$sitename) || !is.null(site.id)) {
-        filename_pattern = paste(filename_pattern, "[^.]*", sep="\\.") #double backslash for regex
-      }
+      #pattern is always the name of the meteorological data product (met).
```

Comment:
1.  Actually has to do with your second change request; see below.

2. Oh, that's an old comment from when I originally modified convert.input; I can take it out.

3. The changes in this pull request only affect the `forecast` section.  The `forecast` section is entered only if forecast=TRUE in the data product's registration xml.  Because NOAA GEFS is the only `forecast` data product we have right now, I'm not sure what else we can test these changes with.